src/perspective_view/panel.ts

```diff
@@ -1054,21 +1059,38 @@ export class PerspectivePanel extends RenderedDataPanel {
           renderContext.depthBufferTexture =
             this.offscreenFramebuffer.colorBuffers[OffscreenTextures.Z].texture;
         }
-        // Draw max projection layers
-        if (
-          renderLayer.isVolumeRendering &&
-          isProjectionLayer(renderLayer as VolumeRenderingRenderLayer)
-        ) {
+        // Draw volume rendering layers
+        if (renderLayer.isVolumeRendering) {
+          const layerIsProjection = isProjectionLayer(
+            renderLayer as VolumeRenderingRenderLayer,
+          );
+          const needTwoRenderingPasses =
+            !layerIsProjection && !this.isCameraMoving;
+          // Draw a single pass if the camera is moving
+          // TODO clearer condition
+          if (!needTwoRenderingPasses && !layerIsProjection) {
+            renderLayer.draw(renderContext, attachment);
+            continue;
+          }
+
           // Set state for max projection mode and draw
-          gl.depthMask(true);
-          gl.disable(WebGL2RenderingContext.BLEND);
-          gl.depthFunc(WebGL2RenderingContext.GREATER);
-          renderContext.emitter = maxProjectionEmit;
-          bindMaxProjectionBuffer();
-          renderLayer.draw(renderContext, attachment);
+          // This happens in the VR layer in two rendering passes
+          if (!needTwoRenderingPasses) {
```

Comment:
So split this out. What happens in VR if it's a 1. Projection layer. 2. A non-projection layer. 3. A projection-layer but the camera is moving (maybe an enum)